doc/source/aggregation_with_dissolve.rst

```diff
@@ -57,3 +57,14 @@ If we are interested in aggregate populations, however, we can pass different fu
 
 .. toctree::
    :maxdepth: 2
+
+### Dissolve Arguments
+
+The ``aggfun =`` argument defaults to 'first' which means that the first row of attributes values found in the dissolve routine will be applied to the resultant dissolved geodataframe.
+However it also accepts other summary statistic options as allowed by ``pandas.groupby()`` including:
+
+* first
```

Comment:
I would make this `'first'` (to make clear it is a string)

Reply:
Hopefully I understand correctly. I made this change:
```
* 'first'
* 'last'
* 'min'
* 'max'
* 'sum'
```